Cmake build failed

### Description

![image](https://github.com/libcpr/cpr/assets/62045004/b42f4acb-4ef3-4f76-9e16-54f58c93a5d2)


### Example/How to Reproduce

1. i had install libssl-dev and libcurl4-openssl-dev
2. run cmake --build . 
3. raised error ：/root/xxx/cpr/cpr/session.cpp:160:58: error: ‘CURLSSLOPT_NATIVE_CA’ was not declared in this scope
  160 |     curl_easy_setopt(curl_->handle, CURLOPT_SSL_OPTIONS, CURLSSLOPT_NATIVE_CA);


### Possible Fix

_No response_

### Where did you get it from?

GitHub (branch e.g. master)

### Additional Context/Your Environment

- OS: Ubuntu 
- Version: 20.04.LTS arm64

--- a/cpr/session.cpp
+++ b/cpr/session.cpp
@@ -163,8 +163,10 @@ void Session::prepareCommon() {
     const bool noRevoke = bitmask & CURLSSLOPT_NO_REVOKE;
 #endif
 
+#if LIBCURL_VERSION_NUM >= 0x074700 // 7.71.0
     // Fix loading certs from Windows cert store when using OpenSSL:
     curl_easy_setopt(curl_->handle, CURLOPT_SSL_OPTIONS, CURLSSLOPT_NATIVE_CA);
+#endif
 
 // Ensure SSL no revoke is still set
 #if SUPPORT_SSL_NO_REVOKE

